src/safe-upgrade-tool/install.md

```diff
@@ -0,0 +1,100 @@
+---
+group: software-update-guide
+title: Install
+ee_only: True
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool ALPHA (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+{:.bs-callout-warning}
+At the moment this is an ALPHA version with limited scope, only validating PHP Magento APIs and GraphQL schema.
+
+## Workflow
+
+The following diagram shows the expected workflow when running the SUT:
+
+![SUT Diagram](img/mvp-diagram.png)
+
+### Who is the SUT for?
+
+The following use case describes the typical process for a Magento partner to upgrade a client's Magento instance:
+
```

Comment:
Let's add this image here
![image](https://user-images.githubusercontent.com/10559423/105175833-bba47680-5b24-11eb-87b5-bd350f8efa41.png)

Reply:
added

Reply:
I can't see it but great that you add it. Thanks